src/v2/email/EmailService.ts

```diff
@@ -0,0 +1,22 @@
+import { MailerService } from '@nestjs-modules/mailer';
+import { Injectable } from '@nestjs/common';
+
+import { EmailOptions } from './dto/EmailDTO';
+
+@Injectable()
+export class EmailService {
+  constructor(private mailerService: MailerService) {}
+
+  async sendEmail({ to, subject, message, link }: EmailOptions) {
+    await this.mailerService.sendMail({
+      to,
+      subject,
+      template: './confirmation',
+      context: {
+        // filling curly brackets confirmation.hbs with content
```

Comment:
Where is the confirmation.hbs file?

Reply:
Changed  to `template.hbs` - default template for email with message and link fields. Located in `src/v2/email/templates`